Build error on latest build using cosmos-theme

Hey there! At https://github.com/CosmWasm/docs2 we are using your theme. The build has broke in the recent changes. I assume its due to banner additions. I've reverted the dependency on older versions. It would be great if you could provide documentation on how to fix this error. Here is the log:
```
2020-07-28T10:57:59.9828722Z > cosmwasm-docs@1.0.0 build /home/runner/work/docs2/docs2
2020-07-28T10:57:59.9829639Z > vuepress build --no-cache
2020-07-28T10:57:59.9829848Z 
2020-07-28T10:58:00.9250876Z wait Extracting site metadata...
2020-07-28T10:58:01.0804548Z tip Apply theme vuepress-theme-cosmos ...
2020-07-28T10:58:01.0814612Z warning [vuepress] Cannot resolve Layout.vue file in 
2020-07-28T10:58:01.0815231Z  /home/runner/work/docs2/docs2/node_modules/vuepress-theme-cosmos
2020-07-28T10:58:01.0815847Z /home/runner/work/docs2/docs2/node_modules/vuepress-theme-cosmos/layouts, 
2020-07-28T10:58:01.0816354Z  fallback to default layout: /home/runner/work/docs2/docs2/node_modules/@vuepress/core/lib/node/theme-api/Layout.fallback.vue
2020-07-28T10:58:01.0977635Z tip Apply plugin container (i.e. "vuepress-plugin-container") ...
2020-07-28T10:58:01.0981273Z tip Apply plugin @vuepress/register-components (i.e. "@vuepress/plugin-register-components") ...
2020-07-28T10:58:01.0986451Z tip Apply plugin vuepress-plugin- ...
2020-07-28T10:58:01.5404717Z tip Clean cache...
2020-07-28T10:58:01.5404961Z 
2020-07-28T10:58:01.9788606Z tip Clean cache...
2020-07-28T10:58:01.9789122Z 
2020-07-28T10:58:02.1970320Z [info] [webpackbar] Compiling Client
2020-07-28T10:58:02.2165559Z [info] [webpackbar] Compiling Server
2020-07-28T10:58:27.9232827Z [success] [webpackbar] Client: Compiled successfully in 25.73s
2020-07-28T10:58:29.8974965Z [success] [webpackbar] Server: Compiled successfully in 27.68s
2020-07-28T10:58:30.0364091Z Error in processing description: $page.content is undefined
2020-07-28T10:58:30.0368608Z Error in processing frontmatter: $page.content is undefined
2020-07-28T10:58:30.0370021Z Error in processing headings.
2020-07-28T10:58:30.0380084Z wait Rendering static HTML...
2020-07-28T10:58:30.1542742Z [Vue warn]: Error in render: "TypeError: Cannot read property 'banner' of undefined"
2020-07-28T10:58:30.1542899Z 
2020-07-28T10:58:30.1543004Z found in
2020-07-28T10:58:30.1543054Z 
2020-07-28T10:58:30.1543462Z ---> <Anonymous>
2020-07-28T10:58:30.1543724Z        <Root>
2020-07-28T10:58:30.2831029Z error Error rendering /404.html: false
2020-07-28T10:58:30.2831169Z undefined
2020-07-28T10:58:30.2862517Z [Vue warn]: Error in render: "TypeError: Cannot read property 'banner' of undefined"
2020-07-28T10:58:30.2862607Z 
2020-07-28T10:58:30.2862710Z found in
2020-07-28T10:58:30.2862759Z 
2020-07-28T10:58:30.2863243Z ---> <Anonymous>
2020-07-28T10:58:30.2863356Z        <Root>
2020-07-28T10:58:30.2863828Z [Vue warn]: Error in render: "TypeError: Cannot read property 'banner' of undefined"
2020-07-28T10:58:30.2864096Z 
2020-07-28T10:58:30.2864186Z found in
2020-07-28T10:58:30.2864255Z 
2020-07-28T10:58:30.2864664Z ---> <Anonymous>
2020-07-28T10:58:30.2864970Z        <Root>
2020-07-28T10:58:30.2865287Z [Vue warn]: Error in render: "TypeError: Cannot read property 'banner' of undefined"
2020-07-28T10:58:30.2865407Z 
2020-07-28T10:58:30.2865804Z found in
2020-07-28T10:58:30.2865862Z 
2020-07-28T10:58:30.2866131Z ---> <Anonymous>
2020-07-28T10:58:30.2866254Z        <Root>
2020-07-28T10:58:30.2873847Z [Vue warn]: Error in render: "TypeError: Cannot read property 'banner' of undefined"
2020-07-28T10:58:30.2874537Z 
2020-07-28T10:58:30.2874662Z found in
2020-07-28T10:58:30.2874737Z 
2020-07-28T10:58:30.2875030Z ---> <Anonymous>
2020-07-28T10:58:30.2875157Z        <Root>
2020-07-28T10:58:30.2884951Z [Vue warn]: Error in render: "TypeError: Cannot read property 'banner' of undefined"
2020-07-28T10:58:30.2885065Z 
2020-07-28T10:58:30.2885174Z found in
2020-07-28T10:58:30.2885226Z 
2020-07-28T10:58:30.2885653Z ---> <Anonymous>
2020-07-28T10:58:30.2885770Z        <Root>
2020-07-28T10:58:30.2897182Z [Vue warn]: Error in render: "TypeError: Cannot read property 'banner' of undefined"
2020-07-28T10:58:30.2897284Z 
2020-07-28T10:58:30.2897407Z found in
2020-07-28T10:58:30.2897465Z 
2020-07-28T10:58:30.2898005Z ---> <Anonymous>
2020-07-28T10:58:30.2898148Z        <Root>
2020-07-28T10:58:30.2909239Z [Vue warn]: Error in render: "TypeError: Cannot read property 'banner' of undefined"
2020-07-28T10:58:30.2909509Z 
2020-07-28T10:58:30.2909799Z found in
2020-07-28T10:58:30.2909870Z 
2020-07-28T10:58:30.2910298Z ---> <Anonymous>
2020-07-28T10:58:30.2910388Z        <Root>
2020-07-28T10:58:30.2918437Z [Vue warn]: Error in render: "TypeError: Cannot read property 'banner' of undefined"
2020-07-28T10:58:30.2918545Z 
2020-07-28T10:58:30.2918650Z found in
2020-07-28T10:58:30.2918697Z 
2020-07-28T10:58:30.2918931Z ---> <Anonymous>
2020-07-28T10:58:30.2919035Z        <Root>
2020-07-28T10:58:30.2926062Z [Vue warn]: Error in render: "TypeError: Cannot read property 'banner' of undefined"
2020-07-28T10:58:30.2926147Z 
2020-07-28T10:58:30.2926432Z found in
2020-07-28T10:58:30.2933538Z 
2020-07-28T10:58:30.2934954Z ---> <Anonymous>
2020-07-28T10:58:30.2935079Z        <Root>
2020-07-28T10:58:30.2935437Z [Vue warn]: Error in render: "TypeError: Cannot read property 'banner' of undefined"
2020-07-28T10:58:30.2935530Z 
2020-07-28T10:58:30.2935612Z found in
2020-07-28T10:58:30.2942194Z 
2020-07-28T10:58:30.2942807Z ---> <Anonymous>
2020-07-28T10:58:30.2942925Z        <Root>
2020-07-28T10:58:30.2943257Z [Vue warn]: Error in render: "TypeError: Cannot read property 'banner' of undefined"
2020-07-28T10:58:30.2943353Z 
2020-07-28T10:58:30.2943461Z found in
2020-07-28T10:58:30.2943692Z 
2020-07-28T10:58:30.2943936Z ---> <Anonymous>
2020-07-28T10:58:30.2944053Z        <Root>
2020-07-28T10:58:30.2952045Z [Vue warn]: Error in render: "TypeError: Cannot read property 'banner' of undefined"
2020-07-28T10:58:30.2952153Z 
2020-07-28T10:58:30.2952241Z found in
2020-07-28T10:58:30.2952311Z 
2020-07-28T10:58:30.2952539Z ---> <Anonymous>
2020-07-28T10:58:30.2952647Z        <Root>
2020-07-28T10:58:30.2963531Z [Vue warn]: Error in render: "TypeError: Cannot read property 'banner' of undefined"
2020-07-28T10:58:30.2963956Z 
2020-07-28T10:58:30.2964363Z found in
2020-07-28T10:58:30.2974668Z 
2020-07-28T10:58:30.2975328Z ---> <Anonymous>
2020-07-28T10:58:30.2975591Z        <Root>
2020-07-28T10:58:30.2976118Z [Vue warn]: Error in render: "TypeError: Cannot read property 'banner' of undefined"
2020-07-28T10:58:30.2976368Z 
2020-07-28T10:58:30.2976573Z found in
2020-07-28T10:58:30.2976718Z 
2020-07-28T10:58:30.2977547Z ---> <Anonymous>
2020-07-28T10:58:30.2978045Z        <Root>
2020-07-28T10:58:30.2979078Z [Vue warn]: Error in render: "TypeError: Cannot read property 'banner' of undefined"
2020-07-28T10:58:30.2979326Z 
2020-07-28T10:58:30.2979537Z found in
2020-07-28T10:58:30.2979728Z 
2020-07-28T10:58:30.2980143Z ---> <Anonymous>
2020-07-28T10:58:30.2980398Z        <Root>
2020-07-28T10:58:30.2995763Z [Vue warn]: Error in render: "TypeError: Cannot read property 'banner' of undefined"
2020-07-28T10:58:30.2996101Z 
2020-07-28T10:58:30.2996369Z found in
2020-07-28T10:58:30.2996573Z 
2020-07-28T10:58:30.2997022Z ---> <Anonymous>
2020-07-28T10:58:30.2997308Z        <Root>
2020-07-28T10:58:30.2998186Z [Vue warn]: Error in render: "TypeError: Cannot read property 'banner' of undefined"
2020-07-28T10:58:30.2998398Z 
2020-07-28T10:58:30.2998607Z found in
2020-07-28T10:58:30.2998781Z 
2020-07-28T10:58:30.2999140Z ---> <Anonymous>
2020-07-28T10:58:30.2999386Z        <Root>
2020-07-28T10:58:30.3003756Z [Vue warn]: Error in render: "TypeError: Cannot read property 'banner' of undefined"
2020-07-28T10:58:30.3004014Z 
2020-07-28T10:58:30.3004221Z found in
2020-07-28T10:58:30.3004373Z 
2020-07-28T10:58:30.3004738Z ---> <Anonymous>
2020-07-28T10:58:30.3004982Z        <Root>
2020-07-28T10:58:30.3013343Z [Vue warn]: Error in render: "TypeError: Cannot read property 'banner' of undefined"
2020-07-28T10:58:30.3013452Z 
2020-07-28T10:58:30.3013541Z found in
2020-07-28T10:58:30.3013610Z 
2020-07-28T10:58:30.3013840Z ---> <Anonymous>
2020-07-28T10:58:30.3013947Z        <Root>
2020-07-28T10:58:30.3020722Z [Vue warn]: Error in render: "TypeError: Cannot read property 'banner' of undefined"
2020-07-28T10:58:30.3020953Z 
2020-07-28T10:58:30.3021073Z found in
2020-07-28T10:58:30.3021220Z 
2020-07-28T10:58:30.3021500Z ---> <Anonymous>
2020-07-28T10:58:30.3021610Z        <Root>
2020-07-28T10:58:30.3026894Z [Vue warn]: Error in render: "TypeError: Cannot read property 'banner' of undefined"
2020-07-28T10:58:30.3026996Z 
2020-07-28T10:58:30.3027097Z found in
2020-07-28T10:58:30.3027146Z 
2020-07-28T10:58:30.3027371Z ---> <Anonymous>
2020-07-28T10:58:30.3027478Z        <Root>
2020-07-28T10:58:30.3050096Z [Vue warn]: Error in render: "TypeError: Cannot read property 'banner' of undefined"
2020-07-28T10:58:30.3050225Z 
2020-07-28T10:58:30.3050346Z found in
2020-07-28T10:58:30.3050403Z 
2020-07-28T10:58:30.3050668Z ---> <Anonymous>
2020-07-28T10:58:30.3050790Z        <Root>
2020-07-28T10:58:30.3051113Z error Error rendering /DOCS_README.html: false
2020-07-28T10:58:30.3051211Z undefined
2020-07-28T10:58:30.3051327Z error Error rendering /: false
2020-07-28T10:58:30.3051488Z undefined
2020-07-28T10:58:30.3078431Z error Error rendering /getting-started/edit-escrow-hints.html: false
2020-07-28T10:58:30.3078577Z undefined
2020-07-28T10:58:30.3078951Z error Error rendering /getting-started/editing-escrow-contract.html: false
2020-07-28T10:58:30.3079085Z undefined
2020-07-28T10:58:30.3079559Z error Error rendering /getting-started/first-demo.html: false
2020-07-28T10:58:30.3079666Z undefined
2020-07-28T10:58:30.3080290Z error Error rendering /getting-started/intro.html: false
2020-07-28T10:58:30.3080409Z undefined
2020-07-28T10:58:30.3080696Z error Error rendering /getting-started/rust-basics.html: false
2020-07-28T10:58:30.3080818Z undefined
2020-07-28T10:58:30.3081305Z error Error rendering /getting-started/smart-contracts.html: false
2020-07-28T10:58:30.3081426Z undefined
2020-07-28T10:58:30.3081686Z error Error rendering /getting-started/using-the-sdk.html: false
2020-07-28T10:58:30.3081805Z undefined
2020-07-28T10:58:30.3081918Z error Error rendering /intro/actor.html: false
2020-07-28T10:58:30.3082027Z undefined
2020-07-28T10:58:30.3082309Z error Error rendering /intro/addresses.html: false
2020-07-28T10:58:30.3082418Z undefined
2020-07-28T10:58:30.3082527Z error Error rendering /intro/composition.html: false
2020-07-28T10:58:30.3082624Z undefined
2020-07-28T10:58:30.3082731Z error Error rendering /intro/multichain.html: false
2020-07-28T10:58:30.3082837Z undefined
2020-07-28T10:58:30.3082944Z error Error rendering /intro/query.html: false
2020-07-28T10:58:30.3083050Z undefined
2020-07-28T10:58:30.3083156Z error Error rendering /intro/roadmap.html: false
2020-07-28T10:58:30.3083262Z undefined
2020-07-28T10:58:30.3083355Z error Error rendering /intro/serialization.html: false
2020-07-28T10:58:30.3083464Z undefined
2020-07-28T10:58:30.3083571Z error Error rendering /media/assets.html: false
2020-07-28T10:58:30.3083676Z undefined
2020-07-28T10:58:30.3083957Z error Error rendering /name-service/intro.html: false
2020-07-28T10:58:30.3084072Z undefined
2020-07-28T10:58:30.3084344Z error Error rendering /testnets/build-requirements.html: false
2020-07-28T10:58:30.3084443Z undefined
2020-07-28T10:58:30.3084550Z error Error rendering /testnets/testnets.html: false
2020-07-28T10:58:30.3084659Z undefined
2020-07-28T10:58:30.3084768Z error Error rendering /tooling/reviews.html: false
2020-07-28T10:58:30.3084875Z undefined
2020-07-28T10:58:30.3084983Z error Error rendering /tooling/verify.html: false
2020-07-28T10:58:30.3085093Z undefined
2020-07-28T10:58:30.3085403Z TypeError: Cannot read property 'banner' of undefined
2020-07-28T10:58:30.3086899Z     at Proxy.render (node_modules/vuepress-theme-cosmos/layouts/GlobalLayout.vue?57e6:1:89)
2020-07-28T10:58:30.3087067Z     at VueComponent.Vue._render (/home/runner/work/docs2/docs2/node_modules/vue/dist/vue.runtime.common.dev.js:3538:22)
2020-07-28T10:58:30.3087422Z     at resolve (/home/runner/work/docs2/docs2/node_modules/vue-server-renderer/build.dev.js:8415:27)
2020-07-28T10:58:30.3088146Z     at waitForServerPrefetch (/home/runner/work/docs2/docs2/node_modules/vue-server-renderer/build.dev.js:8287:3)
2020-07-28T10:58:30.3088576Z     at renderComponentInner (/home/runner/work/docs2/docs2/node_modules/vue-server-renderer/build.dev.js:8426:3)
2020-07-28T10:58:30.3088930Z     at renderComponent (/home/runner/work/docs2/docs2/node_modules/vue-server-renderer/build.dev.js:8383:5)
2020-07-28T10:58:30.3089405Z     at RenderContext.renderNode (/home/runner/work/docs2/docs2/node_modules/vue-server-renderer/build.dev.js:8294:5)
2020-07-28T10:58:30.3089759Z     at RenderContext.next (/home/runner/work/docs2/docs2/node_modules/vue-server-renderer/build.dev.js:2598:23)
2020-07-28T10:58:30.3090104Z     at cachedWrite (/home/runner/work/docs2/docs2/node_modules/vue-server-renderer/build.dev.js:2451:9)
2020-07-28T10:58:30.3090449Z     at renderElement (/home/runner/work/docs2/docs2/node_modules/vue-server-renderer/build.dev.js:8544:5)
2020-07-28T10:58:30.3090771Z     at renderNode (/home/runner/work/docs2/docs2/node_modules/vue-server-renderer/build.dev.js:8296:5)
2020-07-28T10:58:30.3091111Z     at resolve (/home/runner/work/docs2/docs2/node_modules/vue-server-renderer/build.dev.js:8656:7)
2020-07-28T10:58:30.3092278Z     at waitForServerPrefetch (/home/runner/work/docs2/docs2/node_modules/vue-server-renderer/build.dev.js:8287:3)
2020-07-28T10:58:30.3092686Z     at render (/home/runner/work/docs2/docs2/node_modules/vue-server-renderer/build.dev.js:8658:5)
2020-07-28T10:58:30.3093038Z     at Object.renderToString (/home/runner/work/docs2/docs2/node_modules/vue-server-renderer/build.dev.js:9183:9)
2020-07-28T10:58:30.3093369Z     at /home/runner/work/docs2/docs2/node_modules/vue-server-renderer/build.dev.js:9549:22
2020-07-28T10:58:30.8881401Z npm ERR! code ELIFECYCLE
2020-07-28T10:58:30.8882186Z npm ERR! errno 1
2020-07-28T10:58:30.8901289Z npm ERR! cosmwasm-docs@1.0.0 build: `vuepress build --no-cache`
2020-07-28T10:58:30.8901433Z npm ERR! Exit status 1
2020-07-28T10:58:30.8901546Z npm ERR! 
2020-07-28T10:58:30.8901893Z npm ERR! Failed at the cosmwasm-docs@1.0.0 build script.
2020-07-28T10:58:30.8902055Z npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
2020-07-28T10:58:30.8958186Z 
2020-07-28T10:58:30.8958374Z npm ERR! A complete log of this run can be found in:
2020-07-28T10:58:30.8958869Z npm ERR!     /home/runner/.npm/_logs/2020-07-28T10_58_30_890Z-debug.log
2020-07-28T10:58:30.9028523Z ##[error]Process completed with exit code 1.
2020-07-28T10:58:30.9086272Z Post job cleanup.
2020-07-28T10:58:31.0287600Z [command]/usr/bin/git version
2020-07-28T10:58:31.0365739Z git version 2.27.0
2020-07-28T10:58:31.0413514Z [command]/usr/bin/git config --local --name-only --get-regexp core\.sshCommand
2020-07-28T10:58:31.0466435Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'core\.sshCommand' && git config --local --unset-all 'core.sshCommand' || :
2020-07-28T10:58:31.0801292Z [command]/usr/bin/git config --local --name-only --get-regexp http\.https\:\/\/github\.com\/\.extraheader
2020-07-28T10:58:31.0859298Z [command]/usr/bin/git submodule foreach --recursive git config --local --name-only --get-regexp 'http\.https\:\/\/github\.com\/\.extraheader' && git config --local --unset-all 'http.https://github.com/.extraheader' || :
2020-07-28T10:58:31.1208268Z Cleaning up orphan processes
```

--- a/.vuepress/config.js
+++ b/.vuepress/config.js
@@ -13,6 +13,9 @@ module.exports = {
     logo: {
       src: "/logo/SVG/CosmWasm Logo.svg",
     },
+    topbar: {
+      banner: false
+    },
     sidebar: {
       auto: false,
       nav: [

--- a/package.json
+++ b/package.json
@@ -19,7 +19,7 @@
   },
   "homepage": "https://github.com/CosmWasm/docs",
   "dependencies": {
-    "vuepress-theme-cosmos": "1.0.167"
+    "vuepress-theme-cosmos": "^1.0.169"
   },
   "devDependencies": {
     "filehound": "^1.17.4",

